### PR TITLE
[FIX] website_event_excerpt_img: Correct break on mobile view

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

File: website_event_excerpt_img/static/src/css/website_event_excerpt_img.sass [hunk 1]
```diff
@@ -17,3 +17,6 @@
         .event-thumbnail
             .img
                 height: 16em
+    .row
+        height: auto !important
+        min-height: 200px !important
```
Comment (Yajo) [inline website_event_excerpt_img/static/src/css/website_event_excerpt_img.sass]: Is it possible to remove the `!important` parts? It's not a good practice for theme-agnostic modules. Just a more concise selector should be enough.

File: website_event_excerpt_img/views/event.xml [hunk 2]
```diff
@@ -82,6 +82,7 @@
             </div>
         </div>
     </div>
+    <div class="clearfix hidden-md hidden-lg"/>
```
Comment (Yajo) [inline website_event_excerpt_img/views/event.xml]: If the clearfix breaks nothing in md and lg screens, I'd remove the hidden-* classes. Is it possible? (Can't test, no runbot here)

File: website_event_excerpt_img/static/src/css/website_event_excerpt_img.sass (hunk 1 above)
Comment (pedrobaeza) [inline website_event_excerpt_img/static/src/css/website_event_excerpt_img.sass]: This was put by Dani. Do you think it's enough to preserve the layout?

File: website_event_excerpt_img/views/event.xml (hunk 2 above)
Comment (pedrobaeza) [inline website_event_excerpt_img/views/event.xml]: Well, in theory it should work for all sizes, but I haven't tried also.

**Comment by rafaelbn** [comment]
This branch has conflicts that must be resolved

**Comment by pedrobaeza** [comment]
cc @Tecnativa

**Comment by Yajo** [comment]
This seems full of garbage, closing. Reopen if needed please.

**Comment by pedrobaeza** [comment]
@Yajo, this should be included on final PR, so please take what you need taken into account what has been deployed.

**Pull request closed without merging**